dolphin/service.py

```diff
@@ -208,6 +216,29 @@ def periodic_tasks(self, raise_on_error=False):
         ctxt = context.get_admin_context()
         self.manager.periodic_tasks(ctxt, raise_on_error=raise_on_error)
 
+class AlertMngrService(service.Service):
+    """Service object for triggering trap receiver functionalities.
+    """
+
+    def __init__(self):
+        super(AlertMngrService, self).__init__()
+
+        self.manager_class_name = constants.TRAP_RECEIVER_CLASS
+        manager_class = importutils.import_class(constants.TRAP_RECEIVER_CLASS)
+        self.manager = manager_class()
+
+    def start(self):
+        """Trigger trap receiver creation"""
```

Comment:
Are these comments or doc strings? usually doc strings are put under """

Reply:
Doc strings for each functions

Reply:
ok